.github/workflows/experimental.yml

```diff
@@ -54,13 +54,55 @@ jobs:
       DOCKER_IMAGE_TAG: ${{ matrix.version }}
       DOCKER_IMAGE_WEBLOG_TAG: ${{ github.sha }}
       BUILDX_PLATFORMS: linux/amd64
+      MODE: manual
     steps:
+      - name: lib-injection test runner
+        id: lib-injection-test-runner
+        # TODO: change this back to `main`
+        uses: DataDog/system-tests/lib-injection/runner@onboarding-rc-injection
+        with:
+          docker-registry: ghcr.io
+          docker-registry-username: ${{ github.actor }}
+          docker-registry-password: ${{ secrets.GITHUB_TOKEN }}
+          test-script: ./lib-injection/run-manual-lib-injection.sh
 
+  auto-lib-injection-tests:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    strategy:
+      matrix:
+        variant:
+        - library: java
+          weblog-variant: dd-lib-java-init-test-app
+        - library: nodejs
+          weblog-variant: sample-app
+        - library: python
+          weblog-variant: dd-lib-python-init-test-django
+        test-case: 
+          - TC1
+          - TC2
+          - TC3
+          - TC4
+          - TC5
+      fail-fast: false
+    env:
+      TEST_LIBRARY: ${{ matrix.variant.library }}
+      WEBLOG_VARIANT: ${{ matrix.variant.weblog-variant }}
+      DOCKER_REGISTRY_IMAGES_PATH: ghcr.io/datadog
+      DOCKER_IMAGE_TAG: latest
+      DOCKER_IMAGE_WEBLOG_TAG: ${{ github.sha }}
+      BUILDX_PLATFORMS: linux/amd64
+      MODE: auto-${{ matrix.test-case }}
```

Comment:
Same thing... I'm not sure if we could remove the "MODE" env parameter. We know we are running "run-auto-lib-injection.sh" and we also have the variable "test-case" in the environment. To simplify as much as possible the "actions" and the CI workflow